trace-archiver-hdf5/src/main.rs

```diff
@@ -38,19 +31,13 @@ struct Cli {
     consumer_group: String,
 
     #[clap(long)]
-    event_topic: Option<String>,
+    trace_topic: String,
```

Comment:
```suggestion
    topic: String,
```

Reply:
Will this not be confusing once there is also a `control_topic` command line argument?

Reply:
Ah, yes. In that case keep this as `trace_topic`.